app/src/main/java/com/example/hulk/mtindo/Createstore.java

```diff
@@ -114,21 +117,25 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View view) {
                 submitForm();
+                bitmapImage = new Bitmap(image.findViewById(R.id.imageview1);
```

Comment:
remove this line. you don't want to get the bitmap from the image view, but the other way round (see below)